cmd/logical-cluster-manager/main.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 var (
-	resyncPeriod = 4 * time.Second
```

Comment:
Perhaps 30 is the right value for production, but during development/testing a shorter period means we won't have to wait as long.